src/test/java/com/ostapiuk/UserActionsWithDashboardTest.java

```diff
@@ -0,0 +1,36 @@
+package com.ostapiuk;
+
+import com.ostapiuk.business.bo.LogInBO;
+import com.ostapiuk.business.bo.UserActionsBO;
+import com.ostapiuk.business.validator.UserActionsValidator;
+import com.ostapiuk.core.driver.DriverProvider;
+import com.ostapiuk.core.properties.ConfigProperties;
+import com.ostapiuk.core.providers.DataObjectsProvider;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class UserActionsWithDashboardTest extends BaseTest {
+
+    LogInBO logInBO;
+    UserActionsBO userActionsBO;
+    UserActionsValidator userActionsValidator;
+
+    @BeforeMethod
+    public void initializeFields() {
+        DriverProvider.getDriver().get(ConfigProperties.getBaseSecureUrlProperty());
+        logInBO = new LogInBO();
+        userActionsBO = new UserActionsBO();
+        userActionsValidator = new UserActionsValidator();
+    }
+
+    @Test(timeOut = 300000)
+    public void verifyUserActionsWithDashboard() {
+        logInBO.logIn(DataObjectsProvider.getSingleUser().getEmail(), DataObjectsProvider.getSingleUser().getPassword());
+        userActionsBO.createNewDashboard();
+        userActionsValidator.verifyDashboardIsAdded();
```

Comment:
As we discussed during a demo session - we can't create tests using this style. 
I've asked to create separate tests. If there is a bug in Create dashboard functionality - you will not have the ability to get the results about Update/Edit/Delete dashboards.